svgdigitizer/electrochemistry/cv.py

```diff
@@ -183,7 +187,7 @@ def _add_time_axis(self, df):
     def plot(self):
```

Comment:
Do we want to run plot as a doctest here to see that it works now?

Reply:
see #71